src/services/dashboards/widgets/use-widget-frame-props.ts

```diff
@@ -0,0 +1,13 @@
+import type { ComputedRef } from 'vue';
+import { computed } from 'vue';
+
+import type { WidgetProps } from '@/services/dashboards/widgets/config';
+
+export const useWidgetFrameProps:(props: WidgetProps, state) => ComputedRef = (props: WidgetProps, state) => computed(() => ({
```

Comment:
```suggestion
export const useWidgetFrameProps = (props: WidgetProps, state): ComputedRef<below interface here!> => computed(() => ({
```
types look like little bit different in our codes